plugin.js

```diff
@@ -159,6 +168,12 @@ function casPlugin (server, options) {
       username: session.username,
       attributes: session.attributes
     }
+    if (_options.value.sessionCredentialsMappings) {
+      const dotProp = require('dot-prop')
+      for (let mapping of _options.value.sessionCredentialsMappings) {
```

Comment:
Don't use `for..of`. Either use a full `for(;;)` loop or a `.forEach` (in that preference order).